independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/UpdateProjectCommandHandler.java

```diff
@@ -45,6 +45,7 @@ public class UpdateProjectCommandHandler implements QuarkusCommandHandler {
     public static final String REMOVE = "Remove:";
     public static final String UPDATE = "Update:";
     public static final String ITEM_FORMAT = "%-7s %s";
+    public static final String DEFAULT_UPDATE_RECIPES_VERSION = "LATEST";
```

Comment:
Let's not hold this pull request for that but for your next work in this area, could you move this constant with the others in `QuarkusUpdateCommand`? I think it makes more sense to have them all in the same place.